inception-example-imls-data-majority/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/datamajority/DataMajorityNerRecommender.java

```diff
@@ -190,7 +196,9 @@ public double evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
             }
         }
 
-        return (double) correct / (double) testData.size();
+        double accuracy = (double) correct / (double) testData.size();
```

Comment:
Wouldnt we compute the metrics at a central place? Accuracy is defined as `correct / all` and should be independent of the recommender, so you should not need to do that here.

Reply:
Yes, you are right, I just want to set up a result class, so Sibgha can complete her UI. I'm plan to implement the score calculations in a different issue and then use a central place for it.

Reply:
I'd inline this: `result.setDefaultScore((double) correct / (double) testData.size())` and add a comment that accuracy is calculated.